lua/lspconfig/util.lua

```diff
@@ -321,25 +321,28 @@ function M.find_package_json_ancestor(startpath)
   end)
 end
 
-function M.get_active_client_by_ft(filetype)
+function M.get_active_clients_list_by_ft(filetype)
```

Comment:
Why the function name change? get_active_clients_by_id is in core

Reply:
That was my original plan for this, but the name was misleading because it actually only returns a list of client names, and not the client themselves. If we change it to return a table, then we would have to loop again to match the non-active config names. 

I'm okay with either solution.